src/app/layout/Breadcrumb.spec.tsx

```diff
@@ -76,7 +76,8 @@ describe('BreadcrumbItem', () => {
     })
 })
 describe('Breadcrumbs', () => {
-    it('should render crumb components in handle ', () => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-expressions
```

Comment:
I have no idea why this is coming up as an error on lint, but I had to add in this disable 🤔

Reply:
Hehe, this actually looks like a small error in the code. Look at the `,` at line 85. Indentation tipped me of, but it doesn't actually seem to affect the tests tho.

Reply:
💡!  Ha. Interesting that JavaScript will just parse and execute this as an expression and let you place random commas all over the place (so long as you don't have a return statement)